gemfile.d/foreman_bootdisk.rb

```diff
@@ -0,0 +1,3 @@
+# For testing the foreman bootdisk proxmox extensions
+# this gem file is required
+gem 'foreman_fog_proxmox'
```

Comment:
If you know a better solution to run the tests for foreman_bootdisk together with the proxmox extension tests I'm happy to change this.

Reply:
AFAIK gemfile.d is only used in tests and is the currently supported way to install additional dependencies. It's not in the gemspec and not used in packaging so I don't really see the problem.

Reply:
I haven't realized this is not a new dependency. Although I hate what we are doing here (Proxmox is not the only one), that's ok.

Reply:
Honestly, I don't like the `gemfile.d` directory being introduced. I don't see any added value there. Can't we just add this dependency to the main `Gemfile` with something like `if Rails.env.test?`?

Reply:
No, because Gemfile is simply not used. The relation is inverse: foreman adds a dependency on foreman_bootdisk. That means only the gemspec is used. Then all Rake tasks are executed from Foreman.